### ISSUE TITLE
Install parallels tools

### ISSUE THREAD

**Issue by ddiachkov** [AntonioMeireles/coreos-packer#1]
Hello! It would be nice if parallels boxes will have parallels tools installed.
(for example you can share your local code directory with CoreOS and it's containers via parallels shared folder which I think works faster than NFS)

I wrote packer.io [template](https://github.com/ddiachkov/coreos-parallels-packer) to do so. Maybe you can reuse my code: https://github.com/ddiachkov/coreos-parallels-packer/blob/master/scripts/install-parallels-tools.sh

**Comment by AntonioMeireles** [comment]
hi, thanks for the pointer and very sorry for the long lag in my reply. 

I'm not sure  i'd want to get this feature specifically for the CoreOs images for several orders of reasons: 
-  the virtualbox ones also _only_ have NFS and i think that it is _really_ good to have absolute feature parity between the virtualbox images and parallel ones (and VMware's), so IMHO no more, no less features is a good thing.
- also, for very practical reasons i 'd like upstream - aka CoreOS inc to some day support nativelly parallels in their image build pipeline by default and i suspect that in order to achieve that goal avoiding anything that adds extra entropy to the process would be a good thing. (cf. coreos/coreos-vagrant#199)
- third, i find parallels behaviour performance regarding NFS to be good enough for the general case. 
- fourth, Parallels historically not always do a very timelly job assuring that their kmod builds and behaves   as it is expected to in very recent kernels. knowing this and knowing the intrinsic rolling nature of CoreOS this could get _tricky_ fast as one could get easily in a position where the kmod built nicelly for one specific CoreOS version but just won't build for the next. Yes, i know that things eventually get fixed but this would only attract noise and pain imho to our side in the meantime.

Anyway, these are just my initial thoughts. things evolve over time :smile:   

all the best,

**Comment by ddiachkov** [comment]
Good point! You are right, less entropy – less bugs. I think I'll simply add Vagrant's provision block to install parallels tools.
